PsychSourceGL/Cohorts/FTGLTextRenderer/OGLFT.cpp

```diff
@@ -1255,6 +1257,17 @@ QString Face::format_number ( const QString& format, double number ) { return( Q
       return faces_[0].face_->size->metrics.y_ppem;
   }
 
+  double Raster::underline_position ( void ) const
```

Comment:
Don't know if this is needed? Don't the virtual function definitions in the mother class already define those as returning zero? I don't care either way, and haven't done C++ with virtual functions since quite a while, but those might be redundant.

Reply:
The `virtual double underline_position ( void ) const = 0;` doesn't mean return zero, but that its an abstract function that must be defined in a derived class. Hence the need for this here.

Reply:
Ok. My C++ is a bit rusty. Haven't done anything hard-core in over 10 years...